Extensions/magiccards_popups.js

```diff
@@ -0,0 +1,101 @@
+//* TITLE magiccards.info Popups **//
+//* VERSION 0.1.2 **//
+//* DESCRIPTION    Creates visual popups for links to cards on magiccards.info **//
+//* DEVELOPER thebombzen **//
+//* FRAME false **//
+//* BETA true **//
+
+/* globals XKit */
+
+
+/*
+ * magiccards_popups is an extension to New XKit. It creates popup images for links to http://magiccards.info/
+ * Note that it only creates popups for links to direct cards, not searches.
+ * It adds popups on your dashboard and official tumblr pages, but not on user blogs.
+ */
+
+XKit.extensions.magiccards_popups = new Object({
+
+    addPopup: function($atag, image_source){
+        var $img = $("<img>");
+        $img.prop("src", image_source);
+        $img.css({
+            "padding": "0px",
+            "border":  "0px",
+            "width": "312px",
+            "height": "445px",
+            "position": "fixed",
+            "visibility": "hidden",
+            "z-index": "300"
+        });
+        $atag.addClass("magiccards_popups-tooltip");
+        $atag.hover(function(){
+            $(this).children("img").last().css("visibility", "visible");
+        }, function(){
+            $(this).children("img").last().css("visibility", "hidden");
+        });
+        $atag.append($img);
+        $atag.on("mousemove.magiccards_popups", function(event){
+            var x = event.clientX, y = event.clientY;
+            if (y > window.innerHeight / 2){
+                y -= 485;
+            }
+            $(this).children("img").last().css({
```

Comment:
I've had issues with a local anonymous function being unable to reference a local variable from a function that declared it. I'll try your suggested method, and see if that runs into an issue.